.devcontainer/bsim-local/devcontainer.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "Bluetooth w/ babblesim (passthrough workspace)",
+    "build": {"dockerfile": "../Dockerfile"},
+
+    "containerEnv": {
+        "HISTCONTROL": "ignoredups:erasedups",
+        "ZEPHYR_BASE": "/workspaces/zephyr",
+        "ZEPHYR_TOOLCHAIN_VARIANT": "zephyr",
+        "BSIM_OUT_PATH": "/opt/bsim/",
+        "BSIM_COMPONENTS_PATH": "/opt/bsim/components",
+        "EDTT_PATH": "../tools/edtt"
+    },
+
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "ms-vscode.cpptools",
+                "xaver.clang-format",
```

Comment:
isn't the `ms-vscode.cpptools` enough to do the formatting?

Reply:
I don't think it uses clang-format directly. My understanding is that `cpptools` uses the editor's configuration (here provided by the editorconfig plugin) to do the formatting.

I can remove the clang-format extension if you prefer. I would also prefer only having extensions from microsoft if possible.

Reply:
> I can remove the clang-format extension if you prefer.
Yeah, I dont think the clang-format ext is necessary

> I would also prefer only having extensions from microsoft if possible.
I think as long as they are from verified publisher / have large user base, it should be fine

Reply:
`cpptools` comes bundled with clang-format binary and will pick up the configuration from `.clang-format` in my experience.